### PR TITLE
Tweak QuickSight datasets

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

Two minor tweaks:
- remove redundant code in `login_events` (we moved back to a full refresh and forgot to remove the incremental code)
- change field types in `notifications` from `BIT` (which seems to actually be cast to `INTEGER` in QuickSight) to the more correct `BOOLEAN`

# Test instructions | Instructions pour tester la modification

in staging aft (or on dev if we get that sorted out)
- verify in staging that the `login_events` table still has a (full) refresh scheduled daily
- verify the types of the `service_active` and  `service_count_as_live` columns in `notifications`.

# Release Instructions | Instructions pour le déploiement

- notify users that the notification columns `service_active` and  `service_count_as_live` should be treated as boolean rather than integer. For example, filters should be on `service_active = true` rather than `service_active = 1`

# Reviewer checklist | Liste de vérification du réviseur

- [ ] This PR does not break existing functionality.
- [ ] This PR does not violate GCNotify's privacy policies.
- [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
- [ ] This PR does not significantly alter performance.
- [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

**Review by P0NDER0SA — COMMENTED** [review]
👀

File: aws/quicksight/dataset_notifications.tf [hunk 1]
```diff
@@ -127,11 +127,11 @@ resource "aws_quicksight_data_set" "notifications" {
       }
       columns {
         name = "service_active"
-        type = "BIT"
+        type = "BOOLEAN"
       }
       columns {
         name = "service_count_as_live"
-        type = "BIT"
+        type = "BOOLEAN"
```
Comment (jimleroyer) [inline aws/quicksight/dataset_notifications.tf:134]: Oh yes better; that might break some dashboards so we'll need to fix.

**Review by jimleroyer — APPROVED** [review]
LGTM!

**Comment by sastels** [comment]
Followup 😞 
https://docs.aws.amazon.com/quicksight/latest/user/supported-data-types-and-values.html

I don't know where I got the list of available types but clearly it's wrong, any bit or boolean values get turned into integers by quicksight :/

**Pull request merged**